sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -65,6 +65,13 @@ def test_fibonacci():
     assert fibonacci(n).limit(n, S.Infinity) == S.Infinity
     assert lucas(n).limit(n, S.Infinity) == S.Infinity
 
+    assert fibonacci(n).rewrite(sqrt) == \
+        2**(-n)*sqrt(5)*((1 + sqrt(5))**n - (-sqrt(5) + 1)**n) / 5
+    assert fibonacci(n).rewrite(sqrt).subs(n, 10).simplify() == fibonacci(10)
```

Comment:
Expand will be more to the point.

For smaller arguments

```
In [1]: %timeit fibonacci(n).rewrite(sqrt).subs(n, 10).expand()
1000 loops, best of 3: 394 µs per loop
In [2]: %timeit fibonacci(n).rewrite(sqrt).subs(n, 5).expand()
1000 loops, best of 3: 394 µs per loop
```

So, I guess 10 is Ok.